UPGRADE.md

```diff
@@ -1,5 +1,16 @@
 # UPGRADE
 
+## dev-develop
+
+## JMS/Metadata package update to 2.0
+
+To update JMS/Metadata to 2.0 the following public functions were removed:
+
+ - Massive\Bundle\SearchBundle\Search\Metadata\Driver::loadMetadataFromFile
+ - Massive\Bundle\SearchBundle\Search\Metadata\Driver::getExtension
+
+In this case we alse needed to increase the required `php` version to `7.2`.
```

Comment:
`In this case` does not feel right here... I would write something like "Because of the update we had to increase the required php version to 7.2"